### DISCUSSION TITLE
Ability to create flow of flows

### DISCUSSION THREAD

**Discussion by rjammart** [postmanlabs/postman-flows#291] [Ideas]
To improve reusability, it could be interesting to allow to use the "output" of a flow as "input" of another one (or alternatively use an "input" of a flow as the "output" of another one)
It could be done with a new component specifying the "input" flow to use and/or an "output" flow.

**Comment by Joren-Thijs-KasparSolutions** [comment]
This! you could reuse a flow as a block in a different flow
